pad.c

```diff
@@ -2478,6 +2478,7 @@ Perl_pad_push(pTHX_ PADLIST *padlist, int depth)
 #if defined(USE_ITHREADS)
 
 #  define av_dup_inc(s,t)	MUTABLE_AV(sv_dup_inc((const SV *)s,t))
+#  define hv_dup_inc(s,t)	MUTABLE_HV(sv_dup_inc((const SV *)s,t))
```

Comment:
These macro are also defined in `regcomp.c` and `sv.c`, maybe they should be moved to a header.

Reply:
That sounds a good idea. Perhaps I'll tidy that up first as a separate prereq PR

Reply:
See https://github.com/Perl/perl5/pull/20808